x/concentrated-liquidity/types/errors.go

```diff
@@ -154,3 +154,10 @@ type InvalidPriceLimitError struct {
 func (e InvalidPriceLimitError) Error() string {
 	return fmt.Sprintf("invalid sqrt price limit given (%s), should be greater than (%s) and less than (%s)", e.SqrtPriceLimit, e.LowerBound, e.UpperBound)
 }
+
+type ZeroLiquidityError struct {
+}
```

Comment:
If there is nothing to format, we could define this as a regular error with `errors.New(...)` right?